jbpm/jbpm-flow/src/main/java/org/jbpm/workflow/instance/node/TimerNodeInstance.java

```diff
@@ -111,7 +117,19 @@ public String[] getEventTypes() {
 
     @Override
     public void triggerCompleted(boolean remove) {
-        triggerCompleted(Node.CONNECTION_DEFAULT_TYPE, remove);
+        Exception e = new TimerExpiredException();
+        ExceptionScopeInstance esi = getExceptionScopeInstance(e);
```

Comment:
Just a picky one, wont be better to write
```suggestion
       ExceptionScopeInstance esi = (ExceptionScopeInstance) resolveContextInstance(ExceptionScope.EXCEPTION_SCOPE, e);
```
and avoid a method call